hack/update-deps.sh

```diff
@@ -1,4 +1,23 @@
 #!/bin/bash -e
 
+readonly REQUIRED_GLIDE_VERSION="v0.13"
+
+function verify_glide_version() {
+    if ! command -v glide >/dev/null 2>&1; then
+		echo "[FATAL] Glide was not found in \$PATH. Please install version ${REQUIRED_GLIDE_VERSION} or newer."
+        exit 1
+	fi
+
+	local glide_version
+	glide_version=($(glide --version))
+	if [[ "${glide_version[2]}" < "${REQUIRED_GLIDE_VERSION}" ]]; then
```

Comment:
this is string comparison? Dot is 0x2e, i.e. smaller than digits. Hence, I believe this line is correct, even with glide 1.9 and older.

Reply:
This will work with 0.9.0, just checked, so it's fine to have it as is.